src/components/KeyboardShortcuts/index.tsx

```diff
@@ -111,7 +120,7 @@ export class KeyboardShortcutsBase extends React.Component<Props> {
 
       const getCodeLineAnchor = _createCodeLineAnchorGetter({ compareInfo });
 
-      switch (event.key) {
+      switch (compareInfo ? event.key : mapKeyInBrowse(event.key)) {
```

Comment:
I'm not sure if this would be any better, but I was wondering whether, rather than having a `mapKeyInBrowse` function, we could just have a `getKey` function which accepts the `key`, and then internally uses `compareInfo` in `this.props` to check whether we are in browse mode or not. We could even introduce a new function for `isInBrowseMode` which does that check, which could then also be used in `render()`. This would mean we wouldn't have to do the check for `compareInfo` here.

Do you think that would be an improvement?

Reply:
Good idea! I like the way to reuse `isInBrowseMode`. I pushed the update.